students/synowiec_krzysztof/lesson_02_flow_control/lost_card.py

```diff
@@ -0,0 +1,9 @@
+def main():
+    numberOfCards = int(input())
+    sum = numberOfCards * (numberOfCards + 1) // 2
```

Comment:
Variable name `sum` shadows python built-in function `sum()`
https://docs.python.org/3/library/functions.html
Please avoid that in future.